pkg/client/cmd/app.go

```diff
@@ -429,6 +432,85 @@ WARNING:
 	Run: appLogs,
 }
 
+var appAutoScaleSetCmd = &cobra.Command{
+	Use:   "autoscale <name> [flags]",
+	Short: "Set autoscale parameters for the app",
+	Long: `Set auto scaling parameters for the application.
+
+You can configure the autoscale parameters for the application. (i.e. the minimum and maximum number of replicas)
```

Comment:
That help text might be improved, suggestion:

    Set application's autoscaling.

    You can set the lower and upper limit for the number of pods of the application, as well as the
    target CPU utilization to trigger the autoscaler.